components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3737,13 +3750,26 @@ where
                             .set_flags(WriteBatchFlags::TRANSFER_LEADER_PROPOSAL.bits());
                         cmd.mut_admin_request()
                             .set_cmd_type(AdminCmdType::TransferLeader);
-                        cmd.mut_admin_request().mut_transfer_leader().set_peer(from);
+                        cmd.mut_admin_request()
+                            .mut_transfer_leader()
+                            .set_peer(from.clone());
                         self.propose_raft_command(
                             cmd,
                             Callback::None,
                             DiskFullOpt::AllowedOnAlmostFull,
                         );
+                        info!("after propose transfer leader command";
+                            "region_id" => self.fsm.region_id(),
+                            "peer_id" => self.fsm.peer_id(),
+                            "to" => ?from,
+                            "last_index" => self.fsm.peer.get_store().last_index(),
+                        );
                     } else {
+                        info!("do real transfer leader";
```

Comment:
Redundant.

`transfer_leader()` will log it.

Reply:
temp log